db/migrate/20170110015814_create_participant_groups.rb

```diff
@@ -0,0 +1,8 @@
+class CreateParticipantGroups < ActiveRecord::Migration
+  def change
+    create_table :participant_groups do |t|
+      t.references :application_letter, index: true, foreign_key: true
+      t.integer :group
```

Comment:
In the documentation you put these fields as `not null`, but in the database they are currently not.